assignment-4/JavaScript/scripts.js

```diff
@@ -0,0 +1,31 @@
+$('document').ready(onReady)
+function onReady() {   
+$('.readmore').click(readMore) 
+function readMore() {
+   $('.readmore').hide();
+   $('#show-this-on-click').slideDown();
+   $('.readless').show();
+}
+$('.readless').click(readLess);
+function readLess() {
+   $('#show-this-on-click').slideUp();
+   $('.readless').hide();
+   $('.readmore').show();
+}
+$('.learnmore').click(LearnMore);
+function LearnMore() {
+   $('#learnmoretext').slideDown();
+   $('.learnmore').hide();
+}
+// Used .click as my event because submit pertains to forms
+$('.button').click(onSubmit);
+function onSubmit(event) {
+      event.preventDefault();
+}
+}
+// Not sure why, but my code wouldn't work with the declarations separate at the bottom. Not sure why and what proper syntax
```

Comment:
@toskan this could be a rendering issue, and depending on where your script was written.

Reply:
Thank you for all the feedback. I will be mindful of brackets and indentation in future and figure out this rendering issue.